docs/source/index.rst

```diff
@@ -6,12 +6,88 @@
+The challenge for computer simulations is that i) almost every interesting system has its own collective variable description and ii) the implementation
+of collective variables and methods has run efficiently on modern computers, to allow reasonable insights into the observable of interest.
+
+No compromises in usability and speed for enhanced-sampling methods!
 --------------------------------------------------------------------
 
-PySAGES provides an extensible framework to compute collective variables and to perform
-enhanced-sampling molecular dynamics simulations to discover reaction pathways and estimate
-free energies.
+PySAGES addresses these challenges by offering a python interface between highly optimized simulation engines and the researcher to implement collective variables and enhance-sampling methods.
+However, PySAGES already provides an extensible framework to compute collective variables and to perform enhanced-sampling molecular dynamics simulations to discover reaction pathways and estimate free energies.
+Most research objectives are achievable by using these implemented collective variables and methods.
+PySAGES connects these methods automatically to a highly optimized simulation back.
+We support two backends at the moment: `HOOMD-blue <https://glotzerlab.engin.umich.edu/hoomd-blue>`__ and
+`OpenMM <http://openmm.org/>`__.
+Both engines have in common that they offer a python interface to the user and implement the simulation on GPUs for best performance.
+PySAGES interacts with both backends directly on the GPU memory, no copying between GPU and host memory is required.
+This approach allows biased simulations without slowing the backend simulation engines down.
+Still, PySAGES implements all methods and collective variables as pure python for access and modification.
+
+
+Getting Started
+---------------
+
+PySAGES is designed to seamlessly integrate with your existing simulations.
+For an easily accessible start we prepared examples in Google Colab documents, they enable easy access to a working simulation environment including GPU hardware.
+For an example of using PySAGES with HOOMD-blue checkout `this <https://colab.research.google.com/github/SSAGESLabs/PySAGES/blob/main/examples/hoomd-blue/Umbrella_Integration.ipynb>`__ example for simple umbrella integration.
+If you are working with OpenMM we offer `this <https://colab.research.google.com/github/SSAGESLabs/PySAGES/blob/main/examples/openmm/Harmonic_Bias.ipynb>`__ notebook, which explores the Alanine Dipeptide dihedral angle via harmonic biasing.
+Using the notebooks enables you to get started with PySAGES without installation.
+For a productive deployment, we recommend a full installation.
+
+We offer more examples, both Google Colab notebooks, and bare metal, in the PySAGES GitHub repository `examples <https://github.com/SSAGESLabs/PySAGES/tree/main/examples>`__.
```

Comment:
what does "bare metal" mean here?

Reply:
bare metal refers to an implementation directly on the hardware not in a VM like a container or google colab.
But might be confusing, so I reword it.